docs/orchestrating-elastic-stack-applications/elasticsearch/snapshots.asciidoc

```diff
@@ -8,6 +8,8 @@ endif::[]
 [id="{p}-{page_id}"]
 = Create automated snapshots
 
+NOTE: Snapshots are important to ensure that your Elasticsearch indices can be recovered in the event of an accidental deletion, or to migrate data across clusters.
```

Comment:
It's up to you to decide, but here's another suggestion that's a little more direct and concise.
```suggestion
NOTE: Snapshots are essential for recovering Elasticsearch indices in case of accidental deletion or for migrating data between clusters.
```

Reply:
Thanks!